src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -12472,6 +12482,15 @@ bool Unit::CanSwim() const
     return HasUnitFlag(UNIT_FLAG_RENAME | UNIT_FLAG_CAN_SWIM);
 }
 
+float Unit::GetAdvFlyingVelocity() const
+{
+    auto advFlying = m_movementInfo.advFlying;
```

Comment:
1. no auto
2. (not a big deal here) auto doesnt create references, this is a copy

Reply:
Changed

For my personal knowledge, is auto bad ?

Reply:
auto is fine if the type is generically known to the people working (or even reading) the code, or even if you don't really care what the type is, however, it hinders the readability when the type is infrequent to the people navigating through it

Reply:
`auto` is banned in our codestyle (with the exception of container iterators and in places where auto is mandatory such as structured bindings)